doc/release-notes-0.4.0.rst

```diff
@@ -0,0 +1,274 @@
+It contains references and code for Bluetooth Low Energy devices in the nRF52 Series, though development on these devices is not currently supported with the nRF Connect SDK.
+
+Highlights
+**********
+
+* Added support for the new triple mode (LTE-M/NB-IoT/GPS) with ``mfw_nrf9160_0.7.0-29.alpha`` (see :ref:`nrf9160_ug_network_mode` for instructions for changing the network mode)
+* Updated BSD library:
+
+  * Added API supporting GPS
+  * Fixed various issues (see the :ref:`nrfxlib:bsdlib_changelog`)
+
+* Added sample showing how to use the new GPS API (tested on nRF9160 DK v0.8.5)
+* Added sample showing how to download a new image via HTTP
+* Added multi-image support (see :ref:`ug_pm`)
+
+Release tag
+***********
+
+The release tag for the |NCS| manifest repository (|ncs_repo|) is **v0.4.0**.
+Check the ``west.yml`` file for the corresponding tags in the project repositories.
+
+To use this release, check out the tag in the manifest repository and run ``west update``.
+See :ref:`Getting the nRF Connect SDK code <cloning_the_repositories_win>` for more information.
+
+
+Supported boards
+****************
+
+* PCA10090 (nRF9160 DK) - only for LTE samples
+* PCA10056 (nRF52840 Development Kit)
+* PCA10059 (nRF52840 Dongle)
+* PCA10040 (nRF52 Development Kit)
+* PCA10028 (nRF51 Development Kit)
+* PCA63519 (Smart Remote 3 DK add-on)
+
+
+Required tools
+**************
+
+In addition to the tools mentioned in :ref:`gs_installing`, the following tool versions are required to work with the |NCS|:
+
+.. list-table::
+   :header-rows: 1
+
+   * - Tool
+     - Version
+     - Download link
+   * - SEGGER J-Link
+     - V6.40
+     - `J-Link Software and Documentation Pack`_
+   * - nRF5x Command Line Tools
+     - v9.8.1
+     - `nRF5x Command Line Tools`_
+   * - dtc (Linux only)
+     - v1.4.6 or later
+     - :ref:`gs_installing_tools_linux`
+
+
+As IDE, we recommend to use |SES| (Nordic Edition), version 4.16 or later.
+It is available from the following links:
+
+* `SEGGER Embedded Studio (Nordic Edition) - Windows x86`_
+* `SEGGER Embedded Studio (Nordic Edition) - Windows x64`_
+* `SEGGER Embedded Studio (Nordic Edition) - Mac OS x64`_
+* `SEGGER Embedded Studio (Nordic Edition) - Linux x86`_
+* `SEGGER Embedded Studio (Nordic Edition) - Linux x64`_
+
+
+Change log
+**********
+
+The following sections provide detailed lists of changes by component.
+
+nRF9160
+=======
+
+* Added the following samples:
+
+  * :ref:`http_application_update_sample` - demonstrates a basic FOTA (firmware over-the-air) update
+  * :ref:`mqtt_simple_sample` - connects to an MQTT broker and sends and receives data
+  * :ref:`nrf_coap_client_sample` - receives data from a public CoAP server
+  * Simple GPS sample - gets the current position and logs it on UART
+
+* Renamed the :ref:`secure_partition_manager` sample from Secure Boot to Secure Partition Manager
+* Various fixes and updates to the :ref:`nrfxlib:bsdlib` (in nrfxlib):
```

Comment:
Is it correct that all those fixes are in the bsdlib? @evenl

Reply:
Section updated as follows:
![image](https://user-images.githubusercontent.com/11227796/57508998-5d5d4300-7303-11e9-8ecb-8bf5f74c6e75.png)